FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -351,14 +368,168 @@ paths:
+        description: Filter results to only return components with a specific parent component. Can leave null to not filter
+        schema:
+          type: string         
+          format: uuid
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ActiveComponent'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
         403:
-          description: Forbidden - this service will not provide fault information
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      deprecated: false
+      security:
+      - SystemRead:
+        - fraihmwork.system.read
+
+  /component/{uuid}:
+    get:
+      tags:
+      - Components
+      summary: findComponentByUuid
+      operationId: findComponentByUuid
+      parameters:
+      - name: uuid
+        in: path
+        description: uuid
+        required: true
+        schema:
+          type: string
+          format: uuid
+      responses:
+        200:
+          description: OK
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ActiveComponent'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      deprecated: false
+      security:
+      - SystemRead:
+        - fraihmwork.system.read
+
+    put:
+      tags:
+      - Components
+      - Component Updates
+      summary: Update existing properties about a component
+      description: This information allows FRAIHMWORK to track basic information about a system. Any extra or custom fields are expected to always be present even if null values are provided at first. 
```

Comment:
"about a component"

Reply:
Fixed, and scrubbed for all other instances